keel-artifact/src/test/kotlin/com/netflix/spinnaker/keel/artifacts/NpmArtifactSupplierTests.kt

```diff
@@ -36,14 +36,14 @@ internal class NpmArtifactSupplierTests : JUnit5Minutests {
       deliveryConfigName = deliveryConfig.name,
       statuses = setOf(CANDIDATE)
     )
-    val versions = listOf("1.0.0-rc", "1.0.0", "1.0.1-5", "1.0.2-h6", "1.0.3-h7-gc0c60369")
+    val versions = listOf("1.0.0-rc", "1.0.0-rc.1", "1.0.0", "1.0.1-5", "1.0.2-h6", "1.0.3-rc-h7.gc0c60369")
     val latestArtifact = PublishedArtifact(
       name = npmArtifact.name,
       type = npmArtifact.type,
       reference = npmArtifact.reference,
-      version = "${npmArtifact.name}-${versions.last()}",
-      metadata = mapOf("releaseStatus" to CANDIDATE, "buildNumber" to "1", "commitId" to "a15p0")
-      )
+      version = versions.last(),
+      metadata = mapOf("releaseStatus" to CANDIDATE, "buildNumber" to "7", "commitId" to "gc0c60369")
```

Comment:
As far as I can see, the commit id we present is 7 chars (I'm also fixing it in my PR)

Reply:
Yes, you're right, that's the size of a "short hash". I'm not sure why this example had a longer hash, I'll fix it. My reason for changing things here was simply to have consistency between the metadata and the version string.